### ISSUE TITLE
Accessibility feedback for Usability Testing Prep - Facilities [Vet Center pages]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16257]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

VSA: Jennifer
VSP: Trevor

## Design Feedback

**Note:**
It appears research was already completed, and the information here is presenting the findings.

If additional research, whether exploratory or usability, is planned, please collaborate with your VSA accessibility specialist to review the research plan and any additional materials.

Copy edits provided in UXPin.

## Guidance for development

Adding a quote is a new content type, iirc. Please use the semantic element `<blockquote>` for the quotes. | [MDN on blockquote](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)

To include a reference to the source of quoted material which is contained within a `<blockquote>` or `<q>` element, use the cite attribute on the element. | [MDN on cite](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite)

```html
<blockquote>
  "I held it in for 40 years."
  <cite>
    Vet Center Client
  </cite>
</blockquote>
```

![Screen Shot 2020-11-17 at 9 33 43 AM](https://user-images.githubusercontent.com/57469/99403162-254f9c80-28b8-11eb-80bb-a8cb0e35e726.png)

---
Accordion `+` buttons should be vertically centered when we have multiple lines of text. Small detail. Our design system component seems to behave this way by default.

![Screen Shot 2020-11-17 at 9 44 41 AM](https://user-images.githubusercontent.com/934879/99412275-2aade680-28ba-11eb-96fe-88f5c1e25a06.png)
